### ISSUE TITLE
[1.8.19] iPhone 6 / iOS 9.3.2 - Music added even if user not select it - Change music process don't add music and udate lomotif

### ISSUE THREAD

**Issue by hiteshbheda** [Lomotif/lomotif-ios-issues#424]
Frequency: Every time

Action:
1. Launch Lomotif
2. Open any existing lomotif with any music / song added
3. Click on Music icon to change music at the bottom of opened lomotif
4. Click on My Music tab
5. Click on Menu of My Music
6. Select Playlist option
7. Click on any category which has no song (Classic Music, My Top Rated, etc..)
8. On the next screen, Click on Apply button
9. Click on "OK" to message Screen "Are you sure you want to process without adding music?"
10. Check the Music now

Expected result: There should no music as we have processed without music

Actual result: Even we have not added any music and processed lomotif still lomotif contains previous Music / song

**Comment by xiuie** [comment]
@pmtheint Hmm I think what the tester is saying here is that he can't remove music.. Or that the dialogue box for the "Add music" is misleading.

**Comment by hiteshbheda** [comment]
@xiuie @pmtheint Yes @xiuie, I can't remove song when i update existing lomotif without adding any music. 

Or if it works as design then at the last screen Apply button should be disabled.

**Comment by tsukisa** [comment]
@xiuie @hiteshbheda currently you can't remove a song, we have not added that feature yet

**Comment by hiteshbheda** [comment]
@tsukisa Ok
